app/app.js

```diff
@@ -49,7 +50,7 @@ angular.module('portainer').run([
 
 function ping(EndpointProvider, SystemService) {
   let endpoint = EndpointProvider.currentEndpoint();
-  if (endpoint !== undefined && endpoint.Type === 4) {
```

Comment:
type equality checking was good; especially for when we move to typescript. If the types are different, we should explicitly convert to `number`/`string` instead.

Reply:
I do not folow. What do you mean?

Reply:
You converted `===` to `==` in the process